playtron-os-scripts/playtron-os-scripts.spec

```diff
@@ -1,7 +1,7 @@
 Name: playtron-os-scripts
-Version: 0.3.0
+Version: 0.3.1
 Release: 1%{?dist}
-%define ver 0.3.0
+%define ver 0.3.1
```

Comment:
Now that we are using tags, we can actually use the built-in `%{version}` variable. You can remove this line and then do a find and replacement to change `%{ver}` to `%{version}`.